ts/package.json

```diff
@@ -4,5 +4,8 @@
   "workspaces": [
     "packages/*",
     "tests"
-  ]
+  ],
+  "dependencies": {
+    "@solana/web3.js": "^1.70.0"
```

Comment:
Why do you add  "@solana/web3.js": "^1.70.0" at the top ?

Reply:
oh shit was that me lol. sorry